apps/mark-scan/frontend/src/api.ts

```diff
@@ -167,14 +167,33 @@ export const getApplicationDiskSpaceSummary = {
   },
 } as const;
 
+export const getMostRecentPaperHandlerDiagnostic = {
+  queryKey(): QueryKey {
+    return ['getMostRecentPaperHandlerDiagnostic'];
```

Comment:
In a future PR `getMostRecentAccessibleControllerDiagnostic` and `getMostRecentPaperHandlerDiagnostic ` should probably be combined into `getMostRecentDiagnostic(input: { diagnosticType: DiagnosticType })`  and queries invalidated by query param.